### PR TITLE
RCHAIN-791: Fix node exit in terminal (input not visible)

### PR DESCRIPTION
**Pull request by tgrospic**
## Overview
This PR fixes situation when node exits and terminal does not show any typed input. Only after executing `reset` in terminal input become visible.
This bug is manifesting with Bash on Ubuntu 16.04/18.04.

### JIRA ticket
https://rchain.atlassian.net/browse/RCHAIN-791

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: node/src/main/scala/coop/rchain/node/Main.scala [hunk 1]
```diff
@@ -72,14 +72,12 @@ object Main {
...
     * @param options command line options
-    * @param scheduler main scheduler
     * @param console console
     */
   private def startNode(options: commandline.Options)(
       implicit
-      scheduler: Scheduler,
       console: ConsoleIO[Task]
-  ): Unit = {
+  ): Task[Unit] = {
```
Comment (nzpr) [inline node/src/main/scala/coop/rchain/node/Main.scala:80]: Is there a particular need to use Task here or benefit? Or it's more to make code consistent?

**Review by zsluedem — APPROVED** [review]
LGTM

**Review by nzpr — APPROVED** [review]

**Comment by tgrospic** [comment]
bors merge

File: node/src/main/scala/coop/rchain/node/Main.scala (hunk 1 above)
Comment (tgrospic) [reply node/src/main/scala/coop/rchain/node/Main.scala:80]: Yes, we want to compose all of our `Task` programs and only execute `Task#unsafeRunSync` once, when we really want to execute the program.

**Pull request merged**